ingestion/setup.py

```diff
@@ -45,16 +45,15 @@ def get_long_description():
     "boto3~=1.19.12",
     "botocore==1.22.12",
     "avro-python3==1.10.2",
-    "grpcio-tools",
-    "protobuf",
```

Comment:
Are expected to not use Porto if anymore?

Reply:
It is already a dependency of `grpcio-tools`.